pkg/agent/util/net_windows.go

```diff
@@ -647,6 +663,16 @@ func HostInterfaceExists(ifaceName string) bool {
 	return true
 }
 
+// InterfaceIPv4DhcpEnabled returns the Ipv4 DHCP status on the specified interface.
+func InterfaceIPv4DhcpEnabled(ifaceName string) (bool, error) {
+	adapter, err := getAdapterInAllCompartmentsByName(ifaceName)
+	if err != nil {
+		return false, err
+	}
+	ipv4Dhcp := (adapter.flags&0x00000004 != 0)
```

Comment:
Add a comment to explain why use "0x00000004" for DHCP flag? Maybe provides the win32 API link, then we could trace the source of the value.

Reply:
Updated.